basic/sr_gui_hand_calibration/test/test_hand_calibration.py

```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/env python
+import sys
+import os
+import rospy
+import rospkg
+import unittest
+import rostest
+import tempfile
+from mock import patch
+
+from sr_gui_hand_calibration.sr_hand_calibration_model import HandCalibration
+
+from PyQt5.uic import loadUi
+from PyQt5.QtWidgets import QWidget, QApplication
+
+NAME = "test_hand_calibration"
+PKG = "sr_gui_hand_calibration"
+
+
+class TestHandCalibration(unittest.TestCase):
+
+    def setUp(self):
+        self.app = QApplication(sys.argv)
+        self._widget = QWidget()
+        ui_file = os.path.join(rospkg.RosPack().get_path('sr_gui_hand_calibration'), 'uis', 'SrHandCalibration.ui')
+        loadUi(ui_file, self._widget)
+
+        self.mock_calibration_file = tempfile.NamedTemporaryFile()
```

Comment:
There is constructor argument `delete` set it to `False` and delete file in teardown :)